app/templates/index.html

```diff
@@ -42,32 +42,45 @@
         <h1>{{ title }}</h1>
     </div>
 
-    <h1><u>About Us</u></h1>
-    <h2>Aerin Brown</h2>
-    <p>Aerin Brown is an incoming second-year Software Engineering student at McGill University, and attends as a 
-        Schulich Leader Scholar. She began programming at twelve years old to follow her blossoming passion 
-        in robotics, artificial intelligence, and game design. Now, she is part of the McGill Robotics team and works on
-        inverse kinematics and pathfinding algorithms for their Mars rover project. She also is an active member of
-        the McGill Game Development club and the McGill Artificial Intelligence Society. She has participated in two 
-        hackathons: MAISHacks, in which her team won Best Beginner Project, and McGameJam. In the future, she hopes
-        to continue exploring various areas of Software Engineering through internships and freelance projects.</p>
-    <hr>
-    <h2>Add Your Name Here</h2>
-    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis fermentum orci non sapien rhoncus tincidunt. 
-        Mauris eu ligula est. Pellentesque ut mi ornare, lacinia nisi ac, varius mi. Phasellus molestie arcu ac leo 
-        tristique, vel ultrices erat commodo. Quisque rhoncus fringilla magna, interdum feugiat est auctor eu. Nulla
-         lacinia leo non est finibus suscipit. Maecenas ut placerat neque. Pellentesque sagittis, libero ut 
-         scelerisque congue, mauris sem bibendum tortor, id fermentum risus leo ac magna. Sed et nulla non dui 
-         interdum ultrices ut in nunc. Suspendisse potenti. Curabitur quam tortor, posuere sit amet eros eget, 
-         scelerisque consectetur sapien.</p>
-    <hr>
-    <h2>Add Your Name Here</h2>
-    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis fermentum orci non sapien rhoncus tincidunt. 
-        Mauris eu ligula est. Pellentesque ut mi ornare, lacinia nisi ac, varius mi. Phasellus molestie arcu ac leo 
-        tristique, vel ultrices erat commodo. Quisque rhoncus fringilla magna, interdum feugiat est auctor eu. Nulla
-         lacinia leo non est finibus suscipit. Maecenas ut placerat neque. Pellentesque sagittis, libero ut 
-         scelerisque congue, mauris sem bibendum tortor, id fermentum risus leo ac magna. Sed et nulla non dui 
-         interdum ultrices ut in nunc. Suspendisse potenti. Curabitur quam tortor, posuere sit amet eros eget, 
-         scelerisque consectetur sapien.</p>     
+    <div>
```

Comment:
I suggest that you put a class in the div that wraps your section to identify it, that way, in the `main.css` file you can add your custom styles for this section, you can also wrap the individual elements in smaller sections if you want to customize them, this is usually a better practice than using styles directly in the html. 🙌🏻

Reply:
Fixed!